css/default.css

```diff
@@ -61,7 +59,6 @@ a:hover {
 
 #content {
   padding-top: 1em;
-  border-top: 1px dotted #808000;
```

Comment:
I'm going to merge this and then put it back when you aren't looking :yellow_heart: